src/encoded/commands/deploy.py

```diff
@@ -1153,6 +1153,9 @@ def hostname(value):
     # arm arch only available on demo
     if not args.role == 'demo' and args.arm_image_id:
         raise ValueError('Arm architecture is only available on demos')
+    # For AWS_* env vars to work use None instead of 'default'
+    if args.profile_name == 'default':
```

Comment:
Isn't logic like https://github.com/ENCODE-DCC/encoded/blob/ade188d6fd9dd181cf06c184fe6a6a64dc284195/src/encoded/commands/deploy.py#L1077 affected by this?

Reply:
Updated comment.

Reply:
Yeah you're right, I think that's the only spot that's affected by the change here. @lrowe maybe it would be better to set the profile name to `None` when creating the EC2 client here: https://github.com/ENCODE-DCC/encoded/blob/ade188d6fd9dd181cf06c184fe6a6a64dc284195/src/encoded/commands/deploy.py#L395

Reply:
I've adjusted conditions to use `profile_name != 'production'` and made profile_name None by default.